packages/ui-components/Common/Select/SelectScrollButton/index.tsx

```diff
@@ -0,0 +1,130 @@
+import { ChevronDownIcon, ChevronUpIcon } from '@heroicons/react/24/outline';
+import { useEffect, useRef, useState } from 'react';
+import { type FC, type RefObject } from 'react';
+
+import styles from '@node-core/ui-components/Common/Select/index.module.css';
+
+type SelectScrollButtonProps = {
+  direction: 'up' | 'down';
+  selectContentRef?: RefObject<HTMLDivElement | null>;
+  scrollAmount?: number;
+  scrollInterval?: number;
+};
+
+const SelectScrollButton: FC<SelectScrollButtonProps> = ({
+  direction,
+  selectContentRef,
+  scrollAmount = 35,
+  scrollInterval = 50,
```

Comment:
```suggestion
  scrollInterval = 500,
```

for moment it's to quick